packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -201,11 +201,18 @@ const PlanItem: React.FunctionComponent< Props > = ( {
 									disabled={ !! disabledLabel }
 								>
 									<span>
-										{ isSelected ? TickIcon : '' }
-										{ isSelected
-											? sprintf( __( 'Current Selection', __i18n_text_domain__ ), name )
-											: /* translators: %s is a WordPress.com plan name (eg: Free, Personal) */
-											  sprintf( __( 'Select %s', __i18n_text_domain__ ), name ) }
+										{ disabledLabel ?? (
+											<>
+												{ isSelected ? TickIcon : '' }
+												{ isSelected
+													? sprintf( __( 'Current Selection', __i18n_text_domain__ ), name )
+													: sprintf(
+															// translators: %s is a WordPress.com plan name (eg: Free, Personal)
+															__( 'Select %s', __i18n_text_domain__ ),
+															name
+													  ) }
```

Comment:
I'm not sure if this is the same case as p1612463769120800/1612255293.049500-slack-C013QHLF28Y but worth checking. 
cc: @yansern

Reply:
I can rewrite it in a more "explicit" way for peace of mind

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/49523/commits/76c8e35fdd96b8489f60db78c3ad8af8a9c53720